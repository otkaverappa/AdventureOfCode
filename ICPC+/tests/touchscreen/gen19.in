19
xznyuikmitvyjvwnqtk 10
psduboondougnvvlpdx
eoqzobwkqndaexfazuo
mxcigrebdvhcbvtbjwz
vnmynsytsxnglmkvsez
uhxdlayedjddeqredkc
ycbnloaxksblokkrwlr
coafrgvjnbtpbxtpkhr
ktllfayptualwpneiwb
rjemagmvxydoiyzwdbu
vvovimmxqxvuqeydher
cdphragtwjuqerecbsq 10
zlnufebdjikckntshty
cpjwfqqlurgmqrcnwio
brwlviboaiiakzkihcy
uwqaiirkvpsjrlhcgqf
xsphtzgfkphkkdcknlc
yirsukebnmtsjmiqhjz
mvovgyyillvplggeari
dguxaglkwuwvgrmbzmb
kxohmanvgnoqrukqwqd
jpxfmfyyianjkkzrzbf
uhuixnekdccilrfsgkr 10
gsttddgfwfgdzqajpnp
ztrebfxhxdtqkojdtol
ylqfqrxqcmermzlsaqp
jptfiftrkmgwntmsler
cgdiarhljjdztssyaxt
uhhcduxsoiyfmgjugcd
tmnwlirfgrebnniptfm
ltxlbjrmeyohrawqneh
umaanooyfhduuyrhzay
lgycqqemisrpodpptdf
tlowfkwwryyplhonxgs 10
lomedaiurdawwlmuqwq
onqodcxtpuzjipvmuyw
orzoqvzcrrzihoyytcv
msqndacbmwzjlskbiid
nzuojeckcwmxigpykpa
lczmnmgxnpfqefnsoru
atqnqbtgzfxbszagmon
mceruiyjapawpvneoqa
wphtqattghhvtlzngjn
plesiupxadnazeivuko
nqxvmtgnioxvgiaaswp 10
pwvezwihrdrhsjfpxay
niomfuuhwpdoecjjbfr
kzwdgpongonebvtndpj
kmyqafslovshhrgmyxc
ldqzitwdizssmgscgxu
tnrmxaddmdaopfhqpao
sjqndctvecvavkrjhrn
mfqothuazjdoemerrgm
mmqjmoufxdwkqbdgwka
zkmcaqqejhmxvyohkeb
pbgnmyqrfmbfnntpnlh 10
tuqfumdktpowgpetddl
uiyypnljfbumurdanrf
xmxlkdaozgrkabkarym
cdpypjpulcltbzqmjtp
yuvrfxtpymnmoqbmime
caiqtkpjyzenzzksgkn
vizjwpbzejlkonueget
rfuysvzfofpbazdlxup
waajmoxhsfnoxskxngw
swdjzfkerhzifbiqoxn
xrvkhudsrqapkwstvxf 10
aqobytehjugzrzuchoh
bfzbujzmewllyczzase
hdypcspwwwmdzrecnpd
bvzngypfzrxfzagqfyf
budozchdgxujypjnvje
aiwzoxbuniutkoyapdj
ujgothdewanbcxxblue
hhnbczrbbgekaptokct
oytdzvbwyosdwaszetr
fuxlgabboldiacdfdzg
axxucvwuubnngjnsqnv 10
rdgxnizrocqwepvbsqx
onycafnnzfauxgcwtlv
kbzcxgsthkleyydczkq
mlxpfwvhsqunbvpfvvx
qejbikanmbxepicepab
ztutgvrxcmuacylfjvf
wkiwoygseytixmcsuzj
ubxobznnhykouuxsjvb
dbzyjwmlqjmadoxrpzg
eiepwbjtvuqwasvydtl
rmudozrlqjmzpufgtgs 10
memfamzynetavfwbtvs
hndvmvssblatzgighwf
hjkcjhkhidcdmqgkcbc
wfpzyqfixofcvqoacwk
jepmhccqmgtqfafebvj
jsaoxxhnxjjhsqxgzbk
rprlfynmcojnyenoblv
qkgasbsrhrusjjlwqjj
cnalaypqosdmjnulfvd
ydwsxfdkdttpvipjipa
aftdteropyjuxoqrnyv 10
xdossmajvlavlgqpzug
nlezhdnyxbysabitwut
fpginsodhpapfntevzt
vwwtoxwysstnakukzmy
dvpfkusdyneukaqfqpc
phwkvyupiwdizyygisy
ntospyufelviaeemzfg
qnctwdtwjdokrjalyah
ggsdouhvgjapbntvlzq
hkuyunjuakxhqdcvswc
pfnpwpepmsoczzyzvlj 10
sntpvlsxgmvkbayswpx
obrefsdgsbrdtfyicjc
brpwdszblxsknvbsdtv
jnwcsrhqblbemsvlwnm
zajsmxpnpsjmbyleren
hgzlmndhybwkbzwvlvk
bmfmagmlmdqanybykqd
tpfrbiqzdcupfiurjdd
wpjorwmswylzrcgjdoz
ftdblikhbtkfrcogtms
okrzlidstggvncypnjb 10
ukwhpnjftetmudfvppy
hjhpewtelgnodykkpyv
xrzqfwwmtlcsunbltxf
xlnnzqlkackvzdxslto
xozzgvojghhnfubstrf
dsjppjtmbehrbwqdenr
otzxhgrkakbhqtqflba
xeefvidnlhacvudsdll
nlwouokltvmuutaboie
ctmcyjydebppqcnhxbt
kxqytkrtmgcsixemxol 10
buosjfwxotahfxxesjv
mvbqqlpuynkjqexkqei
psepzjpynjijvfmnvyf
syucjmjgwbneqfihhuy
fhipsfxhtsfymfuroga
wcbjitptaynzfwhvpms
whmdhbkbsyhswlvgwox
pqverccyxtnrruguehg
fzgntdaqjwginwftqhw
ofpdyhaedenkloqajvb
zexhongtgwbckisqgbq 10
newcqhrjiaelclkjzxp
teluiyecqkdgzkecalv
lvvqgzbrjaoavuordov
feilmhvqlxdgjzcbfbc
ymenpahdtmrpuxxfjgc
bszeykdallepjsuyhwi
kpudgqafxlmzmebrcnw
fzjlovdktnidzaadgqf
morypfcqygexlfgwvda
gxpqaoqarztzlhqlyxp
pwvvthczfxdhfcwwelm 10
heladyjvlhtaypyvifx
jmvouasqwxdlgiwgnxr
jiacjzthudoumapdwrv
nouszaaxioubxfcbodw
xzzntlpkqmcmbsittjw
qtmnvlszmicihbhwxul
hlzjzcdhywsunmicjtw
ihemplpyhmuuwhwfgyj
qyhiuuwcyfvupcadtnt
twforbvnjenuumbcgyb
jyigtyjtbehwydcmwfj 10
lqpzkjnootopemxlfxw
bbakxynbmmhyxxnykzn
ypjoenadagdwjgztfxi
hmuqkupzubyiznrpteq
zhwedhkcapciyofoazg
zthabjpsyjzokgmolta
qvrsesijiikojfyjhhz
bfkawwiimueomzfefzo
qiycyhiwtrfuslevjad
tozzclbhrgjfzrdbrnl
ogcvcvigsrivlwxkail 10
jcushvlnwdcitkkomhx
uzqexbcuoceblgxeqsp
fruhbnrnedvbxwtdvvf
plhwoseowugozziiayz
qcenfdjygfvlwisuznz
nlvvzxvkfxkhnolavrm
vxtrlpbfkaslpeglfdi
rlfbsvpfvkyjgwczjsa
oedhrsnzfsfnmqvnkqd
rnpqxvozwxtzlzcuswj
seexrqnodzeiquygrtw 10
sstorhptdirnavrxphm
dknitdetmxoipgefxnv
qqfkdffweudsyqfilkm
ewlshazniyafpskbwrg
uxdardqwndibjhmdooe
dwddbsxlvtedqbgqsli
qalzbugnxweczaheezb
rwxwznzifstnkuyjxur
ltnrxoresxfwpbtnbim
jngearaacyuwlplcnat
tutzqiclvfvjoisvklv 10
kntglewxglzcevvfofi
zcpvlefdbqowaeqjrwf
odsphylcezjoaldxyjd
emvujxybgsxnicfyjel
nkmyamjfkkrpodkiohj
pqbpdjtjjfpuubgtdve
khobwchinvrwnhzelkx
wucloxoxqrswbzlfxpm
fcjwawebcrmangcawbo
ttfnqgobnlszsxlpxjv
