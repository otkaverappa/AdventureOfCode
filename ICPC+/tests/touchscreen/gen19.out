vnmynsytsxnglmkvsez 67
ycbnloaxksblokkrwlr 67
mxcigrebdvhcbvtbjwz 68
vvovimmxqxvuqeydher 69
psduboondougnvvlpdx 71
rjemagmvxydoiyzwdbu 71
coafrgvjnbtpbxtpkhr 75
eoqzobwkqndaexfazuo 83
uhxdlayedjddeqredkc 86
ktllfayptualwpneiwb 93
kxohmanvgnoqrukqwqd 51
xsphtzgfkphkkdcknlc 55
zlnufebdjikckntshty 63
dguxaglkwuwvgrmbzmb 65
brwlviboaiiakzkihcy 69
yirsukebnmtsjmiqhjz 75
cpjwfqqlurgmqrcnwio 76
mvovgyyillvplggeari 76
uwqaiirkvpsjrlhcgqf 79
jpxfmfyyianjkkzrzbf 88
ztrebfxhxdtqkojdtol 62
lgycqqemisrpodpptdf 63
ltxlbjrmeyohrawqneh 65
umaanooyfhduuyrhzay 66
jptfiftrkmgwntmsler 68
cgdiarhljjdztssyaxt 69
uhhcduxsoiyfmgjugcd 69
gsttddgfwfgdzqajpnp 73
tmnwlirfgrebnniptfm 74
ylqfqrxqcmermzlsaqp 75
wphtqattghhvtlzngjn 70
lomedaiurdawwlmuqwq 75
orzoqvzcrrzihoyytcv 76
onqodcxtpuzjipvmuyw 78
nzuojeckcwmxigpykpa 80
msqndacbmwzjlskbiid 81
plesiupxadnazeivuko 83
mceruiyjapawpvneoqa 86
lczmnmgxnpfqefnsoru 96
atqnqbtgzfxbszagmon 99
ldqzitwdizssmgscgxu 57
kzwdgpongonebvtndpj 64
pwvezwihrdrhsjfpxay 69
sjqndctvecvavkrjhrn 73
mfqothuazjdoemerrgm 76
niomfuuhwpdoecjjbfr 80
tnrmxaddmdaopfhqpao 81
kmyqafslovshhrgmyxc 82
mmqjmoufxdwkqbdgwka 82
zkmcaqqejhmxvyohkeb 85
uiyypnljfbumurdanrf 70
yuvrfxtpymnmoqbmime 72
rfuysvzfofpbazdlxup 73
waajmoxhsfnoxskxngw 73
tuqfumdktpowgpetddl 74
xmxlkdaozgrkabkarym 77
swdjzfkerhzifbiqoxn 78
vizjwpbzejlkonueget 79
cdpypjpulcltbzqmjtp 82
caiqtkpjyzenzzksgkn 88
bfzbujzmewllyczzase 56
fuxlgabboldiacdfdzg 59
ujgothdewanbcxxblue 63
bvzngypfzrxfzagqfyf 64
budozchdgxujypjnvje 69
hhnbczrbbgekaptokct 77
oytdzvbwyosdwaszetr 77
hdypcspwwwmdzrecnpd 80
aqobytehjugzrzuchoh 81
aiwzoxbuniutkoyapdj 91
ubxobznnhykouuxsjvb 57
kbzcxgsthkleyydczkq 58
ztutgvrxcmuacylfjvf 65
mlxpfwvhsqunbvpfvvx 67
dbzyjwmlqjmadoxrpzg 70
qejbikanmbxepicepab 71
rdgxnizrocqwepvbsqx 73
onycafnnzfauxgcwtlv 75
eiepwbjtvuqwasvydtl 79
wkiwoygseytixmcsuzj 81
hjkcjhkhidcdmqgkcbc 64
qkgasbsrhrusjjlwqjj 70
wfpzyqfixofcvqoacwk 70
rprlfynmcojnyenoblv 72
memfamzynetavfwbtvs 73
hndvmvssblatzgighwf 76
jepmhccqmgtqfafebvj 78
ydwsxfdkdttpvipjipa 78
jsaoxxhnxjjhsqxgzbk 82
cnalaypqosdmjnulfvd 92
qnctwdtwjdokrjalyah 61
phwkvyupiwdizyygisy 65
dvpfkusdyneukaqfqpc 67
ntospyufelviaeemzfg 71
vwwtoxwysstnakukzmy 72
nlezhdnyxbysabitwut 73
ggsdouhvgjapbntvlzq 74
hkuyunjuakxhqdcvswc 75
xdossmajvlavlgqpzug 75
fpginsodhpapfntevzt 77
sntpvlsxgmvkbayswpx 74
hgzlmndhybwkbzwvlvk 76
obrefsdgsbrdtfyicjc 76
ftdblikhbtkfrcogtms 77
wpjorwmswylzrcgjdoz 84
brpwdszblxsknvbsdtv 86
bmfmagmlmdqanybykqd 87
jnwcsrhqblbemsvlwnm 89
tpfrbiqzdcupfiurjdd 92
zajsmxpnpsjmbyleren 97
ukwhpnjftetmudfvppy 57
ctmcyjydebppqcnhxbt 70
otzxhgrkakbhqtqflba 72
xrzqfwwmtlcsunbltxf 72
xozzgvojghhnfubstrf 73
xeefvidnlhacvudsdll 74
hjhpewtelgnodykkpyv 75
nlwouokltvmuutaboie 75
dsjppjtmbehrbwqdenr 76
xlnnzqlkackvzdxslto 92
pqverccyxtnrruguehg 64
psepzjpynjijvfmnvyf 65
fzgntdaqjwginwftqhw 67
mvbqqlpuynkjqexkqei 70
buosjfwxotahfxxesjv 74
syucjmjgwbneqfihhuy 74
wcbjitptaynzfwhvpms 78
whmdhbkbsyhswlvgwox 79
ofpdyhaedenkloqajvb 82
fhipsfxhtsfymfuroga 85
morypfcqygexlfgwvda 55
feilmhvqlxdgjzcbfbc 61
fzjlovdktnidzaadgqf 68
ymenpahdtmrpuxxfjgc 70
teluiyecqkdgzkecalv 73
lvvqgzbrjaoavuordov 75
kpudgqafxlmzmebrcnw 78
newcqhrjiaelclkjzxp 80
bszeykdallepjsuyhwi 81
gxpqaoqarztzlhqlyxp 86
qtmnvlszmicihbhwxul 59
twforbvnjenuumbcgyb 59
qyhiuuwcyfvupcadtnt 62
jmvouasqwxdlgiwgnxr 65
ihemplpyhmuuwhwfgyj 69
xzzntlpkqmcmbsittjw 74
jiacjzthudoumapdwrv 76
hlzjzcdhywsunmicjtw 78
heladyjvlhtaypyvifx 81
nouszaaxioubxfcbodw 89
ypjoenadagdwjgztfxi 55
qiycyhiwtrfuslevjad 60
tozzclbhrgjfzrdbrnl 62
bfkawwiimueomzfefzo 64
bbakxynbmmhyxxnykzn 65
zhwedhkcapciyofoazg 69
lqpzkjnootopemxlfxw 70
hmuqkupzubyiznrpteq 71
qvrsesijiikojfyjhhz 72
zthabjpsyjzokgmolta 77
rnpqxvozwxtzlzcuswj 54
oedhrsnzfsfnmqvnkqd 63
uzqexbcuoceblgxeqsp 63
rlfbsvpfvkyjgwczjsa 65
nlvvzxvkfxkhnolavrm 70
qcenfdjygfvlwisuznz 70
fruhbnrnedvbxwtdvvf 71
jcushvlnwdcitkkomhx 78
vxtrlpbfkaslpeglfdi 80
plhwoseowugozziiayz 81
dwddbsxlvtedqbgqsli 54
rwxwznzifstnkuyjxur 54
qalzbugnxweczaheezb 66
sstorhptdirnavrxphm 70
uxdardqwndibjhmdooe 70
dknitdetmxoipgefxnv 72
ewlshazniyafpskbwrg 72
ltnrxoresxfwpbtnbim 79
qqfkdffweudsyqfilkm 81
jngearaacyuwlplcnat 87
khobwchinvrwnhzelkx 56
nkmyamjfkkrpodkiohj 72
fcjwawebcrmangcawbo 73
pqbpdjtjjfpuubgtdve 75
emvujxybgsxnicfyjel 76
kntglewxglzcevvfofi 81
odsphylcezjoaldxyjd 81
ttfnqgobnlszsxlpxjv 83
zcpvlefdbqowaeqjrwf 94
wucloxoxqrswbzlfxpm 98
